bitnami/influxdb/templates/influxdb/cronjob-backup.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.backup.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: "{{ template "influxdb.fullname" . }}-backup"
+  labels:
+    {{- include "influxdb.labels" . | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.backup.annotations | nindent 4 }}
+spec:
+  schedule: {{.Values.backup.schedule | quote }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels: 
+            {{- include "influxdb.matchLabels" . | nindent 12 }}
+        spec:
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
+          restartPolicy: OnFailure
+          volumes:
+          - name: backup-scripts
+            configMap:
+              name: {{ include "influxdb.fullname" . }}-backup
+              defaultMode: 0755
+          {{- if .Values.backup.upload.google.enabled }}
+          - name: google-cloud-key
+            secret:
+              secretName: {{ .Values.backup.upload.google.serviceAccountSecret | quote }}
+          {{- end }}       
+          - name: {{ include "influxdb.fullname" . }}-backups
+          {{- if .Values.persistence.enabled }}
+            persistentVolumeClaim:
+              claimName: {{ include "influxdb.fullname" . }}-backups
+          {{- else }}
+              emptyDir: {}
+          {{- end }}
+          initContainers:
+          - name: influxdb-backup
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            env:
+              - name: INFLUXDB_ADMIN_USER_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ include "influxdb.secretName" . }}
+                    key: admin-user-password
+            command: 
+              - "/tmp/backup.sh"
+            volumeMounts:
+            - name: {{ include "influxdb.fullname" . }}-backups
+              mountPath: {{ .Values.backup.directory | quote }}
+            - name: backup-scripts
+              mountPath: /tmp/backup.sh
+              subPath: backup.sh
+          containers:
+          - name: influxdb-backup-dummy-container
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            command: 
+              - "/bin/true"
+          {{- if .Values.backup.upload.google.enabled }}
+          - name: gsutil-cp
+            image: google/cloud-sdk:alpine
```

Comment:
This image should come from the **values.yaml** (the same way other images do). You'll have to create a macro (e.g. `"influxdb.backup.image"`) and use sth like:
```yaml
            image: {{ include "influxdb.backup.image" . }}
```

Reply:
On the other hand, please note that, for security reasons, we do not accept any image in Bitnami charts that's not packaged by Bitnami. 
At this moment, we don't have any image including GoogleCloudSDK or Azure CLI. Therefore, we need to request adding these images to Bitnami catalog. In the meantime you can use [Minideb](https://github.com/bitnami/minideb) and install these CLIs on top of it.